From c53b3d1117559a75750170b8b724d9df2c74a743 Mon Sep 17 00:00:00 2001
From: Icarus Sparry <icarus.w.sparry@intel.com>
Date: Mon, 21 Aug 2017 14:50:00 -0700
Subject: [PATCH] Allow guile 2.2 as a valid version

Signed-off-by: Icarus Sparry <icarus.w.sparry@intel.com>
---
 agen5/guile-iface.h | 5 +----
 configure           | 2 +-
 2 files changed, 2 insertions(+), 5 deletions(-)

diff --git a/agen5/guile-iface.h b/agen5/guile-iface.h
index 350ccb3..9fb137d 100644
--- a/agen5/guile-iface.h
+++ b/agen5/guile-iface.h
@@ -9,16 +9,13 @@
 # error AutoGen does not work with this version of Guile
   choke me.
 
-#elif GUILE_VERSION < 201000
+#else
 # define AG_SCM_IS_PROC(_p)           scm_is_true( scm_procedure_p(_p))
 # define AG_SCM_LIST_P(_l)            scm_is_true( scm_list_p(_l))
 # define AG_SCM_PAIR_P(_p)            scm_is_true( scm_pair_p(_p))
 # define AG_SCM_TO_LONG(_v)           scm_to_long(_v)
 # define AG_SCM_TO_ULONG(_v)          ((unsigned long)scm_to_ulong(_v))
 
-#else
-# error unknown GUILE_VERSION
-  choke me.
 #endif
 
 #endif /* MUTATING_GUILE_IFACE_H_GUARD */
diff --git a/configure b/configure
index fcd5d1f..a85ec77 100755
--- a/configure
+++ b/configure
@@ -14198,7 +14198,7 @@ $as_echo "no" >&6; }
 		PKG_CONFIG=""
 	fi
 fi
-  _guile_versions_to_search="2.0 1.8"
+  _guile_versions_to_search="2.2 2.0 1.8"
   if test -n "$GUILE_EFFECTIVE_VERSION"; then
     _guile_tmp=""
     for v in $_guile_versions_to_search; do
-- 
2.14.1

